client/client.go

```diff
@@ -15,88 +15,85 @@
 package client
 
 import (
-	"fmt"
-	"net/http"
-	"net/url"
+	"log"
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client/restclient"
-	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/client/unversioned"
 )
 
-type AppControllerClient struct {
-	*http.Client
-	Root                   *url.URL
-	dependenciesURL        *url.URL
-	resourceDefinitionsURL *url.URL
+type Interface interface {
```

Comment:
I believe it's rather 'go-ish'. One usually refers to it like client.Interface. Go linter suggested me that actually. It used to be ClientInterface, but client.ClientInterface is way too redundant. As for adding 'AppController' it would be more like MS products, and AppControllerClientInterface is too long.